letsdrill.rb

```diff
@@ -1,24 +1,40 @@
 def get_letter_grade(integer)
- 
-  #Put your code here!
 
+  percentageGrade = integer.to_i
+
+  puts case percentageGrade
+  when 90..100
+    return "A"
+  when 80..89
+    return "B"
+  when 70..79
+    return "C"
+  when 60..69
+    return "D"
+  when 0..59
+    return "F"
+  else
+    return "Invalid percentage"
+  end
 end
 
 def shortest_string(array)
 
-  #Put your code here!
+  shortString = array[0]
+  array.each {|testString| testString.length < shortString.length ? shortString = testString : shortString}
```

Comment:
I'd encourage you to do some more digging around in the ruby documentation, i.e. [here](https://ruby-doc.org/core-2.1.0/Enumerable.html).  One of the hallmarks of ruby is the wide variety of built-in methods - for many common cases (including this one), the solution is built right into the language.